### ISSUE TITLE
Gephi is only importing part of my edges

### ISSUE THREAD

**Issue by matiascg** [gephi/gephi#2062]
## Expected Behavior
I currently have 2627 edges in a project im working on. When I import, gephi tells me that as well, before i click on finish import
## Current Behavior
When the import is done, at least 300 edges are missing. These edges are all unique and not duplicates
## Possible Solution

The number imported depends somehow on how I sort them in my excel spreadsheet, sometimes it imports only 200 of the 2627, sometimes more. This happens on every computer on which I tried to import the data. Merge strategy doesnt make a difference
## Context

 Im working on a network analysis of the creator enviroment on youtube, so i need my data to be complete in order to draw conclusions

## Your Environment

* Version used: Gephi 0.9.2
* Java version: newest version
[Youtube Clean.xlsx](https://github.com/gephi/gephi/files/2670033/Youtube.Clean.xlsx)

**Comment by l-marchesi** [comment]
I can confirm this, my friends and I worked for a school project with Gephi and we lost edges as well.
We were able to reduce it from 4165 missing edges to a merely 1401 edges missing when choosing no merging strategy.
However, it seems that there is a platform problem too.
My friend with his Macbook Air seems to loose more edges with the sum-merging strategy:
Out of 31530 he gets 26584 edges. Meanwhile using my Windows notebook (Windows 10, 1803) I get 27365 edges out of the same file, using the same import settings.

@martinfrancois feel free to add to it, if I forgot something.
